plugin/vscode/quick-lint-js/vscode/qljs-document.h

```diff
@@ -120,19 +120,11 @@ class QLJS_Config_Document : public QLJS_Document_Base {
                               Loaded_Config_File* config_file) override;
 
  private:
-  void lint_config_and_publish_diagnostics(::Napi::Env, VSCode_Module&,
-                                           VSCode_Diagnostic_Collection,
-                                           Loaded_Config_File* loaded_config);
-
-  ::Napi::Array lint_config(::Napi::Env env, VSCode_Module* vscode,
-                            Loaded_Config_File* loaded_config) {
-    vscode->load_non_persistent(env);
-
-    LSP_Locator locator(&loaded_config->file_content);
-    VSCode_Diag_Reporter diag_reporter(vscode, env, &locator, this->uri());
-    diag_reporter.report(loaded_config->errors);
-    return std::move(diag_reporter).diagnostics();
-  }
+  Loaded_Config_File* loaded_config_ = nullptr;
```

Comment:
Nit: Ideally the changes related to loaded_config_ would be done in a separate patch. I have some opinions on it. But I'll let this slide...